model-engine/model_engine_server/inference/vllm/requirements.txt

```diff
@@ -1,2 +1,2 @@
-vllm==0.4.0.post1
```

Comment:
what happened to post1?

Reply:
saw this bug in `0.4.0.post1` - https://github.com/vllm-project/vllm/issues/3953